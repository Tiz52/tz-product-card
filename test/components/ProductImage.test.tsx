import React from 'react'
import renderer from 'react-test-renderer'
import { ProductImage, ProductCard } from '../../src/components';
import { product2 } from '../data/product';



describe('ProductTitle', () => {

	test('debe de mostrar el componente correctamente con la imagen personalizado', () => {

		const wrapper = renderer.create(
			<ProductImage img={product2.img}/>
		)

		expect(wrapper.toJSON()).toMatchSnapshot();
		
	});

	test('debe de mostrar el component con la img producto', ()=> {

		const wrapper = renderer.create(
			<ProductCard product={product2}>
				{
					() => <ProductImage/>		
				}
			</ProductCard>
		)

		expect(wrapper.toJSON()).toMatchSnapshot();

	});

});
