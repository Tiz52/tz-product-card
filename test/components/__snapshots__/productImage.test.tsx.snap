// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle debe de mostrar el component con la img producto 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product Image"
    className="productImg undefined"
    src="./coffe-mug.png"
  />
</div>
`;

exports[`ProductTitle debe de mostrar el componente correctamente con la imagen personalizado 1`] = `
<img
  alt="Product Image"
  className="productImg undefined"
  src="./coffe-mug.png"
/>
`;
